setup/scaling/index.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: single
+title:  "Scale Spinnaker"
+sidebar:
+  nav: setup
+---
+
+Spinnaker is a large system, made of multiple (up to 10) microservices, each
+intended to be scaled, restarted, and configured independently. This provides
+operators a great of flexibilty, and Spinnaker the ability to scale to handle
+massive scale (1K+ deployments/day, 10K+ managed machines). However, there is
```

Comment:
Because of the distance between "provides" and "Spinnaker," it gets kinda awkward. Maybe "...and makes it possible for Spinnaker to scale massively(..."
("...scale to handle massive scale" doesn't sound quite right either.)

Reply:
Reworded a little differently